### DISCUSSION TITLE
某些图片必现：IndexError: list index out of range

### DISCUSSION THREAD

**Discussion by hujiachun** [PaddlePaddle/PaddleOCR#13181] [Q&A]
#### 问题描述 / Problem Description
![20240624-141354](https://github.com/PaddlePaddle/PaddleOCR/assets/17762932/1c560d8b-7085-40b3-bbf7-e994e9802e72)

#### 运行环境 / Runtime Environment
- OS:Ubuntu 22.04
- Paddle: paddlepaddle-gpu@2.6.1.post120
- PaddleOCR:2.7.3

**Comment by GreatV** [comment]
try install the `main` branch

**Comment by UserWangZz** [comment]
看起来像是模型和字典没匹配上，试试main分支或者release/2.7分支试试
